tests/test_main.py

```diff
@@ -33,7 +34,12 @@ def assert_pyqt5():
     assert QtCore.QEvent is PyQt5.QtCore.QEvent
     assert QtGui.QPainter is PyQt5.QtGui.QPainter
     assert QtWidgets.QWidget is PyQt5.QtWidgets.QWidget
-    assert QtWebEngineWidgets.QWebEnginePage is PyQt5.QtWebKitWidgets.QWebPage
+    from PyQt5 import Qt
+    if LooseVersion(Qt.PYQT_VERSION_STR) < LooseVersion('5.6'):
```

Comment:
This is not exactly right because WebKit can still be compiled and distributed with Qt 5.6.

In fact, the Qt 5.6 versions that we're building at Continuum for Python 2.7 and Python 3.4 on Windows will come with WebKit instead of WebEngine because WebEngine requires VS 2015.

That's why we added a constant called `WEBENGINE` to `QtWebEngineWidgets` in qtpy 1.0.2: to detect which Qt web backend is currently in use.

Please use that constant here :-)